src/main/java/com/kryptokrauts/aeternity/sdk/service/transaction/impl/TransactionServiceImpl.java

```diff
@@ -303,6 +304,12 @@ private EncodedTx createEncodedTxObject(String signedTx) {
   private PostTransactionResult waitUntilTransactionIsIncludedInBlock(
       PostTransactionResult postTransactionResult) {
     if (postTransactionResult != null) {
+      if (!StringUtil.isNullOrEmpty(postTransactionResult.getRootErrorMessage())) {
```

Comment:
does this mean that postTransactionResult can be null? I thought it never is. if this is the case I am fine with the changes :-)

Reply:
It should never be null unless something is really wrong :) - we'll keep that check to remove the possibility of sdk causing an NPE in this (most probably never occuring) case

Reply:
true xD ... I guess I need some rest^^